shell/common/pointer_data_dispatcher.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef POINTER_DATA_DISPATCHER_H_
+#define POINTER_DATA_DISPATCHER_H_
+
+#include "flutter/runtime/runtime_controller.h"
+#include "flutter/shell/common/animator.h"
+
+namespace flutter {
+
+class PointerDataDispatcher {
+ public:
+  virtual void DispatchPacket(std::unique_ptr<PointerDataPacket> packet,
```

Comment:
Good call on making these pure virtual instead of having a no-op implementation.